src/main/java/duke/parser/ExportLockerSelectCommandParser.java

```diff
@@ -0,0 +1,15 @@
+package duke.parser;
+
+import duke.logic.commands.ExportLockerSelectCommand;
+
+public class ExportLockerSelectCommandParser {
+
+    /**
+     * This function is used to parse the user input for exporting the details as a CSV file.
+     * @param arguments this is the user input string for the tags that the user wants
```

Comment:
the actual parsing should be done here. Like basically in your case you should parse what all fields are being included into export. Have a look at some examples like AddCommandParser/EditCommandParser to understand what i mean

Reply:
edited